include/MainWindow.h

```diff
@@ -421,9 +421,9 @@ class MainWindow : public QMainWindow
         void toggleRuleResultsExpanded(bool checked);
 
         /// Pops up a save dialog for a bash remediation
```

Comment:
since we talk about rule results just above this, let's make sure the comment mentions that these roles are generated for the currently selected profile, not for results